plugins/generator-1.21.1/neoforge-1.21.1/block.definition.yaml

```diff
@@ -137,7 +145,9 @@ templates:
 
   # TX: Wall templates

```

Comment:
Certain block bases contain

`@MODASSETSROOT/models/block/@registryname_inventory.json`

variant of block model that could also be omited if item is disabled